src/library_bootstrap.js

```diff
@@ -15,11 +15,12 @@ assert(Object.keys(LibraryManager.library).length === 0);
 addToLibrary({
   $callRuntimeCallbacks: () => {},
 
-  $ExitStatus__docs: '/** @constructor */',
-  $ExitStatus: function(status) {
-    this.name = 'ExitStatus';
-    this.message = 'Program terminated with exit(' + status + ')';
-    this.status = status;
+  $ExitStatus: class {
+    name = 'ExitStatus';
+    constructor(status) {
```

Comment:
This does more than use arrow functions? Was it part of another PR?

Reply:
Yeah, sorry this should have been part of #22808